### PR TITLE
Added DisMoi Extension section to extension page

### PR DESCRIPTION
**Pull request by wdestin**
Resolve https://github.com/CaptainFact/captain-fact/issues/143

### CODE REVIEW

**Comment by Betree** [comment]
@wdestin Feel free to use things like `Fix CaptainFact/captain-fact#143` in your descriptions, so that the issue gets closed as soon as we merge it (Github's magic :slightly_smiling_face:)

**Review by Betree — COMMENTED** [review]
Image is a bit large (> 1Mb), here's a compressed version that we can used instead:
dismoi-example-min

**Comment by wdestin** [comment]
Fixed.

**Review by Miragide — APPROVED** [review]

File: app/styles/_components/App/app.sass [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #main-container
   flex-grow: 1
   padding-left: $sidebar_width
-  height: calc(100vh - 60px)
+  height: 100%
```
Comment (Betree) [inline app/styles/_components/App/app.sass:4]: This change has an important impact on the video page, and potentially other pages on the site. Could we find a solution that only touches the extension page?

Peek 2021-01-20 19-35

**Comment by wdestin** [comment]
My bad, I didn't need this change. It's OK now.

**Review by Betree — APPROVED** [review]

**Pull request merged**